### DISCUSSION TITLE
TV02 - Schedule,time and timezone

### DISCUSSION THREAD

**Discussion by malwikl** [Koenkk/zigbee2mqtt#16199] [Devices]
Hi all,

I am using several [TV02](https://www.zigbee2mqtt.io/devices/TV02-Zigbee.html) thermostats and am a bit confused about the schedule feature. I cannot figure out how the local time of the thermostat is set or how to influence it.

My schedule looks like this:
<img width="796" alt="image" src="https://user-images.githubusercontent.com/4599729/212631431-7e1e1b3d-d250-4b2f-822d-bb16d870cfd5.png">

But when I check how the actuator works, it looks like this:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/4599729/212631788-05938949-0dc9-4cd0-b797-51691eebf87c.png">

When you compare it, there is a time difference of -7 hours between the schedule and the actual execution in local time. I know the easy way would be to adapt the schedule accordingly, but isn't there a way to either set the local time of the thermostat or have a timezone setting?

Sorry if this is a dump question, but couldn't figure it out from the documentation.

Cheers
MAlWiKl

**Comment by henryschirmer** [comment]
I have the same problem. Does anyone have an idea?

**Comment by git-tail-d** [comment]
th graphic fit to the schedule.

But there is an mistake in the schedule.

00:00/17 can not work. It mean that the temperature should be set to 17 degree untill midnight. A second later the next schedule 07:00/19 will set the thermostat to 19 degree until 7:00 o'clock.

Then the next to schedules are obsolete. You set the same temperatures at other times also to 19 degree.

That mean you set the temperature from 0:00:01 to 17:00 to 19 degree, from 17:00:01 till 20:00 to 18 degree and from 20:00:01 till 24:00 to 17 degree.

That is also in the graph.

When you want set the temparature until 7:00 to 17 degree you must remove the first entry and set the second to 07:00/17

Best Regards
